### DISCUSSION TITLE
WIFI and 5G

### DISCUSSION THREAD

**Discussion by hududed** [NVlabs/sionna#585] [General]
Looking to combine ray tracing simulation to predict path losses and using that information to configure connections between both 5G and WIFI TXs and RXs. Is that possible with Sionna? If yes, is there a example / tutorial for it?

**Comment by faycalaa** [comment]
Hi @hududed ,

The ray tracer simulates the propagation of radio waves between each pair of transmit and receive antennas. It is independent of the communication standard used for signal modulation and demodulation.

Sionna includes implementations for some components of 5G NR, but not for WiFi.
